cypress/cypress/integration/header.spec.ts

```diff
@@ -192,7 +192,8 @@ describe("Header", () => {
           "http://scribe.usc.edu/wp-content/uploads/2021/02/PrimShield_Word_SmUse_Gold-Wh_RGB-1.png",
       },
     });
-    cy.viewport(1200, 700);
+    cy.viewport(700, 565);
```

Comment:
is this change committed on purpose or you were just testing something locally?

Reply:
it was for testing, but I already removed it

Reply:
? I don't see any change back?
![Screen Shot 2021-08-04 at 1 51 18 PM](https://user-images.githubusercontent.com/323901/128253116-89b8ab3d-c9c6-499d-aa6e-9416006776ba.png)

Reply:
I set it back like it's in the main branch